lms/djangoapps/ora_staff_grader/tests/test_serializers.py

```diff
@@ -255,3 +263,172 @@ def test_serializer_output(self):
         assert output_data['oraMetadata'] == OpenResponseMetadataSerializer(self.mock_ora_instance).data
         assert output_data['submissions'] == self.mock_submissions_data
         assert output_data['rubricConfig'] == self.mock_rubric_data
+
+
+@ddt.ddt
+class TestScoreFieldAndSerializer(TestCase):
+    """ Tests for ScoreField and ScoreSerializer """
+
+    def test_field_no_values(self):
+        """ An empty dict passed to the field should return None """
+        assert ScoreField().to_representation({}) is None
+
+    @ddt.data('pointsEarned', 'pointsPossible')
+    def test_field_missing(self, missing_field):
+        """ One missing field should result in the field defaulting to 0 """
+        value = {'pointsEarned': 30, 'pointsPossible': 50}
+        del value[missing_field]
+
+        expectedValue = dict(value)
+        expectedValue[missing_field] = 0
+
+        actualValue = ScoreField().to_representation(value)
+        assert expectedValue == actualValue
+
+    def test_field(self):
+        """ Base serialization behavior for ScoreField """
+        data = {
+            'pointsEarned': 20,
+            'pointsPossible': 40
+        }
+        representation = ScoreField().to_representation(data)
+        assert representation == data
+
+    def test_serializer_no_values(self):
+        """ Passing the ScoreSerializer an empty dict should result in two zeroes for the field """
+        data = ScoreSerializer({}).data
+        assert data == {'pointsEarned': 0, 'pointsPossible': 0}
+
+    def test_serialier(self):
+        """ Base serialization behavior for ScoreSerializer """
+        input = {'pointsEarned': 10, 'pointsPossible': 200}
+        data = ScoreSerializer(input).data
+        assert data == input
+
+    @ddt.data('pointsEarned', 'pointsPossible')
+    def test_serializer_missing_field(self, missing_field):
+        """ One missing field should result in the field defaulting to 0 """
+        value = {'pointsEarned': 30, 'pointsPossible': 50}
+        del value[missing_field]
+
+        expectedValue = dict(value)
+        expectedValue[missing_field] = 0
+
+        actualValue = ScoreSerializer(value).data
+        assert expectedValue == actualValue
+
+
+class TestUploadedFileSerializer(TestCase):
+    """ Tests for UploadedFileSerializer """
+
+    def test_uploaded_file_serializer(self):
+        """ Base serialization behavior """
+        input = MagicMock()
+        data = UploadedFileSerializer(input).data
+
+        expected_value = {
+            'downloadURL': str(input.download_url),
```

Comment:
Nit: we have mixed conventions for capitalization of acronyms (e.g `submissionUuid` vs `downloadURL`). I would prefer we just stick to one; my personal preference is the former (camel-casing just the first letter).

Reply:
I looked at the document and we actually currently have this as download_url??? I personally prefer the latter than the former but I checked the [style guide](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#syntax-and-organization) and it says acronyms shouldn't all be capitalized

Reply:
Gonna put in a ticket to potentially change this in the frontend